Gems/Atom/Tools/MaterialEditor/Code/Source/Viewport/MaterialViewportWidget.cpp

```diff
@@ -66,6 +65,12 @@ namespace MaterialEditor
         : AtomToolsFramework::RenderViewportWidget(parent)
         , m_toolId(toolId)
     {
+        // The viewport context created by AtomToolsFramework::RenderViewportWidget has no name.
+        // Systems like frame capturing and post FX expect there to be a context with DefaultViewportContextName
+        auto viewportContextManager = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
+        const AZ::Name defaultContextName = viewportContextManager->GetDefaultViewportContextName();
+        viewportContextManager->RenameViewportContext(GetViewportContext(), defaultContextName);
```

Comment:
The viewport context created by `AtomToolsFramework::RenderViewportWidget` does have a name. It's given a unique name in `ViewportContextManager::CreateViewportContext` called from `RenderViewportWidget::InitializeViewportContext`.

Renaming the viewport context to `DefaultViewportContextName` will conflict in the future when there are more than one viewport opened at the same time, either with other Material Editor viewports or the main editor viewport. We don't have to fix this now, but it's important to keep it in mind.

If there are systems that are expecting a specific name for the viewport context, those will need to be improved to work with any viewport name.

Reply:
This part of the change is restoring code that existed previously. I only realized it was broken while restoring the smoke test script. There needs to be a condition to test for the default or first viewport for this and the msaa function. Needs to be addressed for anything with multiple viewports. Will discuss with @NicholasVanSickle @dmcdiarmid-ly @rgba16f for a follow up change.